templates/email/base_template_en.html

```diff
@@ -0,0 +1,77 @@
+                                        <table style="border-collapse:collapse;border-spacing:0;font-family:HelsinkiGrotesk,Helvetica,Arial,sans-serif;padding:0;text-align:left;vertical-align:top;width:100%">
+                                            <tr style="font-family:HelsinkiGrotesk,Helvetica,Arial,sans-serif;padding:0;text-align:left;vertical-align:top">
+                                                <th style="Margin:0;color:#000;font-family:HelsinkiGrotesk,Helvetica,Arial,sans-serif;font-size:18px;font-weight:400;line-height:1.5;margin:0;padding:0;text-align:left">
+                                                    <img src="https://hds.hel.fi/static/assets/logo/helsinki-fi.svg"
+                                                         alt="Helsinki"
+                                                         style="-ms-interpolation-mode:bicubic;clear:both;display:block;font-family:HelsinkiGrotesk,Helvetica,Arial,sans-serif;height:4rem;margin-bottom:40px;margin-top:56px;max-width:100%;outline:0;text-decoration:none;width:auto">
+                                                    <p class="text-body"
+                                                       style="Margin:0;Margin-bottom:32px;color:#000;font-family:HelsinkiGrotesk,Helvetica,Arial,sans-serif;font-size:16px;font-weight:400;line-height:1.5;margin:0;margin-bottom:32px;padding:0;text-align:left">
+                                                        Venepaikkavaraukset<br>
+                                                        Fredriksberg A 1<br>
+                                                        00100 Helsinki</p>
+                                                    <hr class="rule-thin"
+                                                        style="background:#b2b2b2;border:none;height:1px;margin-bottom:24px;margin-top:24px">
+                                                    <p class="text-sm"
+                                                       style="Margin:0;Margin-bottom:32px;color:#000;font-family:HelsinkiGrotesk,Helvetica,Arial,sans-serif;font-size:14px;font-weight:400;line-height:1.75;margin:0;margin-bottom:32px;padding:0;text-align:left">
+                                                        &copy; Copyright 2020 &bull; All rights reserved &bull;</p>
+                                                </th>
+                                            </tr>
+                                        </table>
+                                    </th>
+                                </tr>
+                            </tbody>
+                        </table>
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+    </center>
+</body>
+</html>
```

Comment:
Nitpick: could you add the newlines at the end of the files pls? 🙏

Reply:
✅